docs/how-tos/continuous_integration.md

```diff
@@ -0,0 +1,142 @@
+# Continuous Integration Continuous Deployment
```

Comment:
In general documentation feels more relatable with "we and our" instead of "You and your"